lib/manager/gitlabci/utils.ts

```diff
@@ -0,0 +1,13 @@
+// !reference [.setup, script]
+const re = /!reference \[\.\w+?(?:, \w+?)\]/g;
+
+/**
+ * Replaces GitLab reference tags before parsing, because our yaml parser doesn't support that.
```

Comment:
```suggestion
 * Replaces GitLab reference tags before parsing, because our yaml parser cannot process them anyway.
```

Maybe this is clearer?